modules/dkan/dkan_dataset/modules/dkan_dataset_rest_api/dkan_dataset_rest_api.module

```diff
@@ -35,3 +35,184 @@ function dkan_dataset_rest_api_services_request_postprocess_alter($controller, $
     }
   }
 }
+
+/**
+ * Implements hook_services_resources_alter().
```

Comment:
I think we should include some kind of comments explaining how and why we're changing the service callback